File: 02.Algorithms/03.Searching/01.Linear Search/linear-search.py
def linearSearch(arr, target):
    for i in range(len(arr)):
        if arr[i] == target:
            return i
    return -1


if __name__ == "__main__":
    arr = [12, 52, 69, 78, 23, 5, 6, 8, 4, 64]
    r = linearSearch(arr, 64)
    if (r == -1):
        print("Element is not present in array")
    else:
        print(f"Element is present at index: {r}.")

File: 02.Algorithms/03.Searching/01.Linear Search/linear-search.java
public class Search {
    public static void main(String[] args) {
        int arr[] = {12, 52, 69, 78, 23, 5, 6, 8, 4, 64};
        int r = linearSearch(arr, 69);
        if (r == -1) {
            System.out.println("Element is not present in array");
        } else {
            System.out.println("Element is present at index " + r);
        }
    }

    public static int linearSearch(int[] arr, int target) {
        for (int i = 0; i < arr.length; i++) {
            if (arr[i] == target) {
                return i;
            }
        }
        return -1;
    }
}
